### DISCUSSION TITLE
全屏播放视频时暂停，视频会下移（有弹幕），同时伴随只要视频底栏自动隐藏后，弹幕会消失的bug

### DISCUSSION THREAD

**Discussion by hanwuhanhan** [the1812/Bilibili-Evolved#4796] [提问 / Q&A]
关闭Bilibili-Evolved后就恢复正常了，这个bug触发没找到规律，但感觉有50%的概率出现，不知道是因为哪个选项冲突了，麻烦大佬们找下问题在哪
![使用的插件1](https://github.com/the1812/Bilibili-Evolved/assets/105441093/cdfe8caa-f75c-4c31-8928-aa3843b740d4)
![使用的插件2](https://github.com/the1812/Bilibili-Evolved/assets/105441093/9bd80eac-d820-4f7e-aa9f-1ca3c536b9f6)
![暂停时视频下移但有弹幕](https://github.com/the1812/Bilibili-Evolved/assets/105441093/0fdd0432-f099-4e23-abcd-d2247eb66b30)
![播放时没有弹幕](https://github.com/the1812/Bilibili-Evolved/assets/105441093/557ce900-2e41-4143-a783-21259345ae4d)

**Comment by Geeyun-JY3** [comment]
`隐藏视频标题层`？参见：#4710

**Reply by hanwuhanhan** [reply]
感谢！
